tests/integration_tests/default_value/main.go

```diff
@@ -273,420 +288,584 @@ func modifyColumnDefaultValueDDL2(ctx context.Context, db *sql.DB) {
+			true,
 		},
-
+		/*
+			{
+				"alter table test.`%s` add column v1 datetime(5) default ? %s",
+				"2020-10-10 10:10:10.9999",
+				ddlDefaultValueFunc,
+				true,
+			},
+			{
+				"alter table test.`%s` add column v1 timestamp(5) default ? %s",
+				"2020-10-10 10:10:10.9999",
+				ddlDefaultValueFunc,
+				true,
+			},
+			{
+				"alter table test.`%s` add column v1 time(5) default ? %s",
+				"10:10:10.9999",
+				ddlDefaultValueFunc,
+				true,
+			},
+		*/
 		// numeric data type
 		{
 			// default bit[1]
-			AddFmt:       "alter table test.`%s` add column v1 bit not null",
-			DefaultValue: []byte{0x01},
-		},
-		{
-			AddFmt:       "alter table test.`%s` add column v1 tinyint not null",
-			DefaultValue: -13,
+			"alter table test.`%s` add column v1 bit default ? %s",
+			[]byte{0x01},
+			ddlDefaultValueFunc,
+			true,
 		},
 		{
-			AddFmt:       "alter table test.`%s` add column v1 mediumint not null",
-			DefaultValue: -13,
-		},
-		{
-			AddFmt:       "alter table test.`%s` add column v1 int not null",
-			DefaultValue: -13,
-		},
-		{
-			"alter table test.`%s` add column v1 bigint not null",
+			"alter table test.`%s` add column v1 tinyint default ? %s",
 			-13,
+			ddlDefaultValueFunc,
+			true,
 		},
+		/*
+			// normal cases, we may transfer to test-infra to reduce cost
+			{
+				"alter table test.`%s` add column v1 mediumint default ? %s",
+				-13,
+				ddlDefaultValueFunc,
+				true,
+			},
+			{
+				"alter table test.`%s` add column v1 int default ? %s",
+				-13,
+				ddlDefaultValueFunc,
+				true,
+			},
+			{
+				"alter table test.`%s` add column v1 bigint default ? %s",
+				-13,
+				ddlDefaultValueFunc,
+				true,
+			},
+		*/
 		{
-			"alter table test.`%s` add column v1 decimal(5) not null",
+			"alter table test.`%s` add column v1 decimal(5) default ? %s",
 			-13,
+			ddlDefaultValueFunc,
+			true,
 		},
 		{
-			"alter table test.`%s` add column v1 float not null",
+			"alter table test.`%s` add column v1 float default ? %s",
 			-13.13,
+			ddlDefaultValueFunc,
+			true,
 		},
 		{
-			"alter table test.`%s` add column v1 double not null",
+			"alter table test.`%s` add column v1 double default ? %s",
 			-13.13,
+			ddlDefaultValueFunc,
+			true,
 		},
-		{
-			"alter table test.`%s` add column v1 bit(4) not null",
-			[]byte{0x03},
-		},
-		{
-			"alter table test.`%s` add column v1 tinyint(4) unsigned not null",
-			13,
-		},
-		{
-			"alter table test.`%s` add column v1 mediumint(4) unsigned not null",
-			13,
-		},
-		{
-			"alter table test.`%s` add column v1 int(4) unsigned not null",
-			13,
-		},
-		{
-			"alter table test.`%s` add column v1 bigint(4) unsigned not null",
-			13,
-		},
-		{
-			"alter table test.`%s` add column v1 decimal(5,2) unsigned not null",
-			13.13,
-		},
-		{
-			"alter table test.`%s` add column v1 float(5,2) unsigned not null",
-			13.13,
-		},
-		{
-			"alter table test.`%s` add column v1 double(5,2) unsigned not null",
-			13.13,
-		},
-
+		/*
```

Comment:
Can you add some comments as to why you are commenting them out? Or we could just remove it?

Reply:
I already add some comments at the begining.